wallet/src/de/schildbach/wallet/ui/UpholdTransferActivity.kt

```diff
@@ -155,6 +158,7 @@ class UpholdTransferActivity : InteractionAwareActivity() {
             }
 
             override fun onTransfer() {
+                transactionMetadataViewModel.markAddressAsTransferInAsync(receiveAddress.toBase58())
```

Comment:
maybe it is better, as stated with the Liquid code, that this should be done after freshAddress is called.

Reply:
Is it possible that the created address isn't used?

Reply:
Yes.  If the user does not finish the transfer or if the transfer fails due to an error from Uphold.